formulas/wick-base/functions/wick-add-config-section

```diff
@@ -27,9 +30,18 @@ wickAddConfigSection() {
         wickRemoveConfigSection "$1" "$2" "${3-}"
     fi
 
-    {
-        echo "${3:-#} BEGIN wickAddConfigSection: $2"
-        cat
-        echo "${3:-#} END wickAddConfigSection: $2"
-    } >> "$1"
+    if [[ -n "$top" ]]; then
```

Comment:
Possible alternate implementation

```
oldContent=$(cat "$1")
section=$(
    echo "${3:-#} BEGIN wickAddConfigSection: $2"
    cat
    echo "${3:-#} END wickAddConfigSection: $2"
)

{
    if [[ -z "$top" ]]; then
        echo "$oldContent"
        echo "$section"
    else
        echo "$section"
        echo "$oldContent"
    fi
} > "$1"
```

Does this make it any easier to see whats going on?